assets/questions/mostCommonChar.js

```diff
@@ -56,19 +76,30 @@ globalData.questions['common'] = {  // eslint-disable-line dot-notation
       input: 'apoiuytrewqsdf*&^%$#ba',
       allowedOutputs: ['a']
     }, {
-      input: 'abb4562873ba',
-      allowedOutputs: ['b']
+      input: 'aBBB4562873ba',
+      allowedOutputs: ['B']
     }, {
       input: 'babaabb',
       allowedOutputs: ['b']
     }],
-    buggyOutputTests: [],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.lowercaseOnly',
+      messages: [
+        'Did you consider lowercase letters only?'
+      ]
+    },
+    {
+      buggyFunctionName: 'AuxiliaryCode.lettersOnly',
+      messages: [
+        'Did you consider alphabetic characters only?'
+      ]
+    }],
     performanceTests: []
   }, {
     instructions: [
```

Comment:
This still doesn't seem like it should be a separate task. Just consolidate it into the first task.